### ISSUE TITLE
Support for azurerm_redis_cache - Primary connection string attribute

### ISSUE THREAD

**Issue by robinmanuelthiel** [terraform-providers/terraform-provider-azurerm#5856]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

In contrast to the Azure Portal, `azurerm_redis_cache` does not expose a Primary connection string attribute. My use case would be to directly inject the connection string into Azure Key Vault.

![image](https://user-images.githubusercontent.com/5874465/75138326-3cc37580-56ea-11ea-99cf-e51ea98b114f.png)

Please be aware, that this Primary connection string seems to be specifically dedicated to the StackExchange.Redis .NET library, so let's consider that vendor-specific attribute for the discussion!

My current workaround is building this connection string by myself:

```hcl
resource "azurerm_key_vault_secret" "redis_cache_connection_string" {
  name         = "RedisCacheConnectionString"
  value        = "${azurerm_redis_cache.default.hostname}:${azurerm_redis_cache.default.ssl_port},password=${azurerm_redis_cache.default.primary_access_key},ssl=True,abortConnect=False"
  key_vault_id = azurerm_key_vault.default.id
}

```

### New or Affected Resource(s)

* azurerm_redis_cache

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by brunhil** [comment]
I am working on this at the moment, will get a PR out early next week! Want to make sure I have the formatting correct before pushing.

**Comment by tombuildsstuff** [comment]
Fixed via #5958
